nano/node/json_handler.cpp

```diff
@@ -1574,6 +1574,41 @@ void nano::json_handler::block_create ()
 			{
 				if (work == 0)
 				{
+					// Difficulty calculation
+					if (!request.get_optional<std::string> ("difficulty").is_initialized ())
```

Comment:
Could use `count` as it would be more clear and shorter.

Reply:
Thanks!